portal/server/controllers/utilisation-report-service/utilisation-report-download/index.js

```diff
@@ -1,18 +1,17 @@
 const stream = require('stream');
 const api = require('../../../api');
-const { getApiData } = require('../../../helpers');
 
 const getReportDownload = async (req, res) => {
   const { userToken } = req.session;
   const { bankId, _id } = req.params;
 
-  const fileData = await getApiData(api.downloadUtilisationReport(userToken, bankId, _id), res);
+  const { data, headers } = await api.downloadUtilisationReport(userToken, bankId, _id);
 
-  res.set('Content-Disposition', fileData.headers['Content-Disposition']);
-  res.set('Content-Type', fileData.headers['Content-Type']);
+  res.set('Content-Disposition', headers['content-disposition']);
+  res.set('Content-Type', headers['content-type']);
```

Comment:
In one of the api tests above, you use `'content-disposition'` and `'content-type'` for the headers instead of the capitalised versions. Which one are you expecting to use?